prospero-metadata/src/main/java/org/wildfly/prospero/metadata/ProsperoMetadataUtils.java

```diff
@@ -53,6 +53,8 @@ public class ProsperoMetadataUtils {
 
     private static final String WARNING_MESSAGE = "WARNING: The files in .installation directory should be only edited by the provisioning tool.";
 
+    public static final String CANDIDATE = "candidate_artifact_channels.txt";
```

Comment:
Could you move this to PrepareCandidateAction please? This file is candidate-specific